### DISCUSSION TITLE
Need help for accessing the Follow Turtlebot exercise

### DISCUSSION THREAD

**Discussion by Arcane-01** [JdeRobot/RoboticsAcademy#2398] [Q&A]
Hello! I wanted to try out the Follow Turtlebot exercise. I followed the instructions provided [here](https://jderobot.github.io/RoboticsAcademy/exercises/Drones/follow_turtlebot). I have setup docker and pulled the latest docker image. However, after creating a docker container with ```docker run --rm -it -p 7164:7164 -p 2303:2303 -p 1905:1905 -p 8765:8765 -p 6080:6080 -p 1108:1108 -p 7163:7163 jderobot/robotics-academy```  and then going to the `127.0.0.1:7164/` address on my browser, I couldn't find the Follow Turtlebot exercise among the available exercises. Additionally, I observed that the "Drone Cat and Mouse" exercise is also not present. 
Are these no longer available in the current release?

**Comment by jmplaza** [comment]
Hi @Arcane-01 ,

which RADI are you using 3.X (=based on ROS 1 Noetic) or 4.X (=based on ROS2 Humble)?

**Reply by Arcane-01** [reply]
Hello @jmplaza,
I had pulled the latest docker image as mentioned in the instructions with `docker pull jderobot/robotics-academy:latest` and after inspecting the image I found that it has `"ROS_DISTRO=noetic", "IMAGE_TAG=3.4.28"`.

**Reply by jmplaza** [reply]
I don't know the current status of that exercise. The drone exercises were recently refactored and migrated to ROS2 Humble and Aerostack2 ([along GSoC-2023](https://theroboticsclub.github.io/gsoc2023-Pawan_Wadhwani/Coding-period-week14/)). @pawanw17 ? @pariaspe @iamarkaj ? For sure it is not migrated yet to Unibotics.

AFAIK the Drone Cat and Mouse is currently broken, as it involves to agents simultaneously (the cat with the user code) and its migration to Aerostack2 is not finished yet.

**Reply by pariaspe** [reply]
Hi, the exercise is not migrated to ROS2 Humble, but it should be working on noetic. No idea why it's not available in the exercise list

**Reply by Arcane-01** [reply]
Hi @jmplaza @pariaspe,
I pulled a new docker image but still couldn't find the exercises I was looking for among the available ones. I haven't been able to resolve this on my own, so would it be alright if I raise a GitHub issue about it?

**Reply by pariaspe** [reply]
Hi, I'm sorry to say that the exercise is not working neither on ROS Noetic. I've checked and the exercise is not migrated to react templates. Feel free to open an issue about it
